### ISSUE TITLE
Author field still "lazy loading" even after being explicitly added to fields

### ISSUE THREAD

**Issue by guillochon** [andycasey/ads#77]
## Expected Behavior
If specified in the `fl=` attribute, a field should not be lazy-loaded when it is accessed.

## Current Behavior
The `author` field appears to still be lazy-loaded, even if it is passed in `fl=` (the warning message regarding lazy-loading still appears).

## Possible Solution
It's possible the field is not being lazy-loaded even though the message indicates this is so.

## Steps to Reproduce (for bugs)
1. Form a query with the author field specified in `fl=`
```
allpapers = ads.SearchQuery(bibcode='2011ApJ...727..107G',
            fl=['id', 'bibcode', 'author'])
```
2. Attempt to access the `author` field, e.g.
```
for paper in allpapers:
    print(paper.author)
```

## Context
If the warning message is factual, the API limit is being reached much more quickly than it should be. Also, the number of queries to the server is being doubled, halving the speed of the script.

## Your Environment

* Python 3.5

**Comment by vsudilov** [comment]
Having trouble reproducing in python 3.4, using the latest code from `master`. Is this a version issue? We aren't doing a good job with tacking releases/version numbers to date.

(FWIW `print(paper.author[0])` prints `González-Gaitán, S.` re #76)

```
In [3]: allpapers = ads.SearchQuery(bibcode='2011ApJ...727..107G',
   ...:             fl=['id', 'bibcode', 'author'])

In [4]: for paper in allpapers:
   ...:         print(paper.author)
   ...:     
/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#snimissingwarning.
  SNIMissingWarning
/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
[u'Gonz\xe1lez-Gait\xe1n, S.', u'Perrett, K.', u'Sullivan, M.', u'Conley, A.', u'Howell, D. A.', u'Carlberg, R. G.', u'Astier, P.', u'Balam, D.', u'Balland, C.', u'Basa, S.', u'Fouchez, D.', u'Guy, J.', u'Hardin, D.', u'Hook., I. M.', u'Pain, R.', u'Pritchet, C. J.', u'Regnault, N.', u'Rich, J.', u'Lidman, C.']

In [5]: 
```

**Comment by guillochon** [comment]
I'm using the version available for install via pip, 0.12.1. Were changes made between that release and the master that would produce this?

**Comment by vsudilov** [comment]
Yep, seems like that's what's going on. Can repro both your issues with latest code from pip.

Going to close this issue and the other and open a discussion re how to improve our handling of releases. Please re-open if your issues don't go away when using the code in `master`.
